cluster/gce/configure-vm.sh

```diff
@@ -428,6 +428,10 @@ function create-salt-kubelet-auth() {
     mkdir -p /srv/salt-overlay/salt/kubelet
     (umask 077;
       cat > "${kubelet_kubeconfig_file}" <<EOF
+{% set api_servers = "http://localhost:8080" -%}
```

Comment:
it appears that that env. variable is never actually populated.  (this is why this PR took soooo long to debug, I had the same reasoning...)